tests: CAD unit test failing Chrome

```
views/connect_another_device
render
with a Fx desktop user that is signed in
shows the marketing area, logs appropriately ‣
AssertionError: expected false to be true

```
## Chrome

![image](https://cloud.githubusercontent.com/assets/128755/23684677/562330b0-036e-11e7-8535-eb22a945e358.png)

## FF

![image](https://cloud.githubusercontent.com/assets/128755/23684683/5f77d490-036e-11e7-9b2a-0b93092a687c.png)

cc @shane-tomlinson

--- a/app/tests/spec/views/connect_another_device.js
+++ b/app/tests/spec/views/connect_another_device.js
@@ -66,6 +66,7 @@ define(function (require, exports, module) {
       describe('with a Fx desktop user that is signed in', () => {
         beforeEach(() => {
           sinon.stub(user, 'isSignedInAccount', () => true);
+          windowMock.navigator.userAgent = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:55.0) Gecko/20100101 Firefox/55.0';
 
           return view.render()
             .then(() => {

--- a/app/tests/spec/views/mixins/sync-auth-mixin.js
+++ b/app/tests/spec/views/mixins/sync-auth-mixin.js
@@ -71,6 +71,8 @@ define(function (require, exports, module) {
 
       beforeEach(() => {
         windowMock.location.origin = ORIGIN;
+        windowMock.navigator.userAgent = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:55.0) Gecko/20100101 Firefox/55.0';
+
         relier.set({
           utmCampaign: 'campaign',
           utmContent: 'content',

